receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -63,6 +71,12 @@ type contribResponse struct {
 	page         int
 }
 
+type cveResponse struct {
+	cves         []getRepoCVEsRepositoryVulnerabilityAlertsRepositoryVulnerabilityAlertConnection
```

Comment:
This type should be shorter now after being renamed? Maybe we just renamed the nodes, and if so, I think we should rename this to be shorter/simpler.

Reply:
See my other comment on this as well.